jupiter-tests/src/test/java/org/junit/jupiter/params/ParameterizedInvocationNameFormatterTests.java

```diff
@@ -323,6 +323,62 @@ void mixedTypesOfArgumentsImplementationsAndCustomDisplayNamePattern() {
 
 	}
 
+
+	@Nested
+	class replacesNonPrintableCharacters {
+
+
+@Test
+void replacesNonPrintableCharactersInArguments() {
+var formatter = formatter(ARGUMENTS_PLACEHOLDER, "enigma");
+
+Arguments args = arguments(
+		"\t", "\r", "\r\n", "\n", "\u200B"
+		);
+
+		assertEquals("\\t, \\r, \\r\\n, \\n, ?", format(formatter, 1, args));
```

Comment:
is this kind of the behavior expected to see? @sbrannen

Reply:
Please do not @-mention me from your repository, and please stop working on JUnit issues that already assigned to somebody else.

If you continue to behave like this, I will be forced to report and/or block you.

Reply:
Yes of course im sorry. I was not aware that there is already an PR for this.